docs/sphinx/users/index.txt

```diff
@@ -121,6 +121,7 @@ Visualization and analysis applications
     imago/index
     iqm/index
     macnif/index
+    micromanager/index
```

Comment:
I already raised this but I can see an argument that acquisition and visualization are equivalent and it seems a bit unnecessary to make a new subheading for one item. Do we expect to have anything else to add here?